vega-settings

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env sh
+# Launch the Python settings app installed under share/vegastrike
+
+# Resolve this script (follow symlinks)
+SELF="$0"
+while [ -h "$SELF" ]; do
+  dir="$(cd -P "$(dirname "$SELF")" && pwd)"
+  SELF="$(readlink "$SELF")"
+  [ "${SELF#/}" = "$SELF" ] && SELF="$dir/$SELF"
+done
+
+BINDIR="$(cd -P "$(dirname "$SELF")" && pwd)"
+PREFIX="$(cd "$BINDIR/.." && pwd)"
+
+APP="$PREFIX/share/vegastrike/python/settings_app/settings_app.py"
```

Comment:
How are we managing the Python dependencies like Kivy? Does this need to setup a virtual environment? Or do we need to have that as a dependency on the package ?

Reply:
I don't know .. we can always fix in another PR